src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,46 @@
 
 class Animal {
   // write your code here
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = false;
+    this.addBeast(this);
```

Comment:
should be only in the `Animal` class constructor (actually, you've added it on the 10 line). If it exists in the constructor of the inherited class it will work for Herbivore as well.